packages/edit-site/src/components/layout/index.js

```diff
@@ -272,7 +256,7 @@ export default function Layout() {
 						className="edit-site-layout__sidebar-region"
 					>
 						<AnimatePresence>
-							{ showSidebar && (
+							{ canvasMode === 'view' && (
```

Comment:
This change makes the sidebar take the whole screen in small viewports for pages and templates, while before it showed the content area. Note that showing the sidebar is the same behavior that navigation, styles, and patterns have in `trunk` right now. I think it's fine to go with this. The previous logic based on listing/editor pages was hard to follow and created inconsistencies.

Reply:
I agree with this remark, I think the previous logic was too complex but I do believe it's better to show the list of "pages" and "templates" directly (not the sidebar), I believe we should try to follow-up with this. Maybe the router needs to be aware of viewport and change the areas accordingly.

Reply:
Yeah, agree. We should show the content area instead.